### PR TITLE
Update variant creator instructions and new screenshots

### PR DESCRIPTION
**Pull request by karolkielecki**

### CODE REVIEW

**Review by fowczarek — COMMENTED** [review]
We should use `PNG` instead of `JPEG`.

File: docs/dashboard/catalog/products.md [hunk 1]
```diff
@@ -286,47 +286,65 @@ Based on these attributes, you can create variants of your product.
...
 In the Select Values tab, make your selection from the previously set up values assigned to each attribute. Once you make your choice, Saleor cross-matches selected values of attributes and creates appropriate product variants.
 
-Choose the variants of the product that you will stock and click _Next_ to move to the Price and SKU tab.
+Choose the variants of the product that you will stock and click _Next_ to move to the Prices and SKU tab.
+
+![Product Variant Creator](../screenshots/catalog-product-variant-attributes.jpg)
+
+:::note
+
+The variant creator page appears only when you create variants for the first time for a newly created product.
```
[the remark this answers is not in the document]
Comment (karolkielecki) [reply docs/dashboard/catalog/products.md:299]: When I was testing the dashboard, the 3-step variant creator used to appear only once. For sure, we should write additional info to our current copy to clarify when exactly it appears.

File: docs/dashboard/catalog/products.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ title: Products
 
 This page contains an alphabetically sorted list of all your products with a picture for each (if set), the product name and type, information on its visibility on your site (if it is published or not), and the product's price.
 
-![](../screenshots/catalog-product-list.png)
+![](../screenshots/catalog-product-list.jpg)
```
[the remark this answers is not in the document]
Comment (karolkielecki) [reply docs/dashboard/catalog/products.md:9]: Ok, I will replace the screenshot

File: docs/dashboard/discounts/sales.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ Sales are an excellent way to encourage customers to use your store. Saleor offe
 
 Click _Create&nbsp;Sale_ above the main sales list and fill in the relevant sections.
 
-![Creating sale](../screenshots/discount-new.jpeg)
+![Creating sale](../screenshots/discount-sale-new.jpg)
```
[the remark this answers is not in the document]
Comment (karolkielecki) [reply docs/dashboard/discounts/sales.md:19]: IMO, it's unnecessary to show at least 2 channels in the screenshot. But, I will add a sentence to the instructions that you can assign more than one channel.

File: docs/dashboard/discounts/vouchers.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ title: Vouchers
 
 Vouchers allow you to give all your customers or selected shoppers a great deal on a specific product or order. Users get a voucher code via email, or on social media or your website, which they can they activate at the checkout when making a purchase.
 
-![Voucher page](../screenshots/discount-voucher-details.jpeg)
+![Voucher page](../screenshots/discount-voucher-details.jpg)
```
[the remark this answers is not in the document]
Comment (karolkielecki) [reply docs/dashboard/discounts/vouchers.md:9]: Same here - IMO, it's unnecessary to show at least 2 channels in the screenshot. But, I will add a sentence to the instructions that you can assign more than one channel.

File: docs/dashboard/catalog/products.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (karolkielecki) [reply docs/dashboard/catalog/products.md]: The note has been updated.

**Comment by fowczarek** [comment]
Should we fix all `jpg` introduced in this PR?

**Comment by karolkielecki** [comment]
I will replace jpg screenshots in this PR. @fowczarek

File: docs/dashboard/catalog/products.md [hunk 5]
```diff
@@ -286,41 +286,59 @@ Based on these attributes, you can create variants of your product.
...
 Once saved, you can edit variants through the same steps as described above and then save changes.
 
-#### Checking inventory
+### Checking inventory
 
-In this example, you can see five types of t-shirt. Four have enough stock to cover future orders. The S size has a red mark and is unavailable. When we click on the product, we can see in the product variant details that the product is unavailable because there are fourteen items in stock (“Inventory”) and they have been ordered by a customer (“Allocated”).
+In this example, when you click on the product, we can see in the product variant details page that the product is still available because there are one hundred thirty eight (138) items in the stock (the _Quantity_ section) and they have been ordered sixty nine (69) times by the customers so far (the _Allocated_ section).
 
-![Checking inventory](../screenshots/catalog-product-stock.jpeg)
+![Checking inventory](../screenshots/catalog-product-checking-inventory.jpg)
```
Comment (fowczarek) [inline docs/dashboard/catalog/products.md:339]: Still `jpg`

**Review by fowczarek — APPROVED** [review]

File: docs/dashboard/catalog/products.md (hunk 5 above)
Comment (karolkielecki) [reply docs/dashboard/catalog/products.md:339]: In this case, I prefer to leave jpg. As far as I can see, there is no such example in our current dashboard (number of allocated products, etc.).

**Review by patrys — APPROVED** [review]

**Pull request merged**